### DISCUSSION TITLE
Restoring locally -> input device is "not a TTY"

### DISCUSSION THREAD

**Discussion by BramvdnHeuvel** [prodrigestivill/docker-postgres-backup-local#94] [Q&A] [answered]
I am running a PostgreSQL container with the name ` Synapse_Database`. It has a database called `synapse` which I reach as user `synapse`. Upon running the command below, the program exits immediately.

```
zcat local-backup.sql.gz | docker exec --tty --interactive Synapse_Database_Backup psql --username=synapse --dbname=synapse -W
```

Instead of seemingly doing anything, it only outputs `the input device is not a TTY`.

I'm running the command on WSL 2 if it matters. Is this a bug or am I doing something wrong?

**Comment by visini** [comment, marked as answer]
Perhaps related: https://github.com/prodrigestivill/docker-postgres-backup-local/issues/54

**Comment by prodrigestivill** [comment]
Yes, this is an slightly different case, but I updated the readme. Thanks for the report.
